src/main/java/br/com/maykmoreira/apiexample/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package br.com.maykmoreira.apiexample.service;
+
+import br.com.maykmoreira.apiexample.model.Movie;
+import br.com.maykmoreira.apiexample.repository.MovieRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MovieServiceImpl implements MovieService {
```

Comment:
If you change the interface to start with "I" then you don't need to add the "Impl" at the end of the implemented class.